src/core/flags.h.in

```diff
@@ -119,4 +134,25 @@
 #cmakedefine USE_ZLIB
 #endif
 
+/* functions to query flags at runtime instead of compile time */
+#ifdef __cplusplus
+extern "C" {
```

Comment:
This goes around the whole file if needed. You should also include `mgba-util/dllexports.h`